src/components/TabsList/TabsList.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  selectedId: string;
+}
+
+export const TabsList: React.FC<Props> = ({ tabs, selectedId }) => {
+  return (
+    <ul>
+      {tabs.map(tab => (
+        <li
+          key={tab.id}
+          data-cy="Tab"
+          className={cn({
+            'is-active': selectedId === tab.id,
+          })}
+        >
+          <Link to={`/tabs/${tab.id}`}>
```

Comment:
you can do like this
```suggestion
          <Link to={`../${tab.id}`}>
```